README.md

```diff
@@ -90,7 +90,7 @@ cases are implemented for each database:
 |CrateDB|X||
 |InfluxDB|X|X|
 |MongoDB|X|
-|QuestDB|X|X
```

Comment:
We support ingestion for the IoT use case while query generation is not supported. Probably it's worth clarifying here.